core/src/main/scala/net/ruippeixotog/scalascraper/browser/JsoupBrowser.scala

```diff
@@ -42,25 +47,25 @@ class JsoupBrowser(val userAgent: String = "jsoup/1.8", val proxy: JavaProxy = n
     executePipeline(Jsoup.connect(url).method(POST).proxy(proxy).data(form.asJava))
 
   def parseFile(file: File, charset: String): JsoupDocument =
-    JsoupDocument(Jsoup.parse(file, charset))
+    JsoupDocument(Jsoup.parse(file, charset, baseUri, parser))
 
   def parseString(html: String): JsoupDocument =
-    JsoupDocument(Jsoup.parse(html))
+    JsoupDocument(Jsoup.parse(html, baseUri, parser))
```

Comment:
Given the above, the `baseUri` you defined as a variable is not always the one used... can you please inline this instead of creating a variable? Inlining an empty string shouldn't be a big deal for DRY.

Ditto for the input stream version.

```suggestion
    JsoupDocument(Jsoup.parse(html, "", parser))
```

Reply:
done